test/cuda/test_execution_engine_cache.cc

```diff
@@ -21,43 +21,42 @@
 #include <glog/logging.h>
 #include <gtest/gtest.h>
 
```

Comment:
execution_engine does not exist anymore, rename this file?

Reply:
just killing it, doesn't test anything useful anymore